external-crates/move/crates/move-unit-test/src/test_reporter.rs

```diff
@@ -110,6 +112,53 @@ impl FailureReason {
     }
 }
 
+fn clever_error_line_number_to_loc(test_plan: &TestPlan, vm_error: &VMError) -> Option<Loc> {
```

Comment:
Small nit,  but some of this functionality feels more general than the test reporter. Would it be worth migrating parts of it to `move-compiler/src/diagnostics/mod.rs`, possibly to live on the `PositionInfo` trait? I could see the IDE or similar benefitting from the ability to turn a line number into its definition loc , etc.

Reply:
Will do!